src/mail-app/settings/MailExportViewer.ts

```diff
@@ -29,12 +29,13 @@ export class MailExportViewer implements UpdatableSettingsViewer {
 			m(
 				".fill-absolute.scroll.plr-l.pb-xl",
 				m(".h4.mt-l", lang.get("mailExportSettings_label")),
-				this.mailExportController
-					? m(MailExportSettings, {
-							mailboxDetails: mailLocator.mailboxModel.mailboxDetails(),
-							logins: mailLocator.logins,
-							mailExportController: this.mailExportController,
-					  })
+				isDesktop()
+					? this.mailExportController &&
```

Comment:
I would rewrite this `&&` as a ternary, mithril doesn't like `undefined`, we are just lucky that `this.mailExportController` is `null`